utils.py

```diff
@@ -0,0 +1,26 @@
+import torch
+import torch.nn as nn
+
+def cross_entropy_mse(pred_win_prob, neural_net_prob, self_play_winner, search_prob, parameters, c = 1.00):
```

Comment:
Not sure we need the regularization included here. It's probably easier to handle that through weight_decay in the optimizer.

Reply:
Got it, haven't really used Pytorch a whole lot other than the 1013 Project, so I don't really know the parameters of other functions, will look into it.